src/librustc_session/options.rs

```diff
@@ -875,13 +875,16 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         `hir,typed` (HIR with types for each node),
         `hir-tree` (dump the raw HIR),
         `mir` (the MIR), or `mir-cfg` (graphviz formatted MIR)"),
-    run_dsymutil: Option<bool> = (None, parse_opt_bool, [TRACKED],
+    // The default historical behavior was to always run dsymutil, so we're
+    // preserving that temporarily, but we're likely to switch the default
+    // soon.
```

Comment:
does this deserve a FIXME or a tracking issue?

Reply:
Perhaps? The comment was pre-existing, I just moved it, so I don't really know.